website/docs/understanding_ferretdb.md

```diff
@@ -115,9 +115,8 @@ To reference the `city` field in the embedded document, use the dot notation `"c
 
 For dot notation examples on embedded documents, see here:
 
-* Query an embedded document
-* Update an embedded document
-* Query an array of embedded documents
+* [Query an embedded document](./basic_operations/read.md#query-on-an-embedded-or-nested-document)
+* [Update an embedded document](./basic_operations/update.md#update-an-embedded-document)
```

Comment:
And here we have `.md` in both places. What's the approach to it? When we do and don't add `.md` to the links?